tools/stress-cluster/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/powershell
+
+RUN pwsh -c Install-Module Az -Force; \
+    pwsh -c Install-Module Az.Accounts -Force;
```

Comment:
Not really a problem, but this would be installed with `Az`, which is just a (large) list of dependent modules.